devicon.json

```
@@ -3856,6 +3856,24 @@
         "color": "#3C99D4",
         "aliases": []
     },
+    {
+        "name": "webassembly",
+        "tags": [
+            "web-environment"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain",
```

Please watch out for commas. JSON is a stricter than JS objects regarding this.
```suggestion
                "plain"
```